packages/validation/src/validators/LionWebReferenceValidator.ts

```diff
@@ -22,6 +22,12 @@ export class LionWebReferenceValidator {
         this.validationResult = validationResult
     }
 
+    /**
+     * Vhecks for dup;licate node id's.
+     * As a side-effect creates a map from `id` to `node` to make further validation faster.
+     * @param obj
```

Comment:
Empty `@param`s are pretty useless: I’d delete them. Also applies to other 2 added JSDocs.